docs/data/sdks/android-kotlin/migration.md

```diff
@@ -1,13 +1,247 @@
+
+To migrate to `com.amplitude:analytics-android`, update your dependencies and instrumentation.
+
+## Terminology
+
+* `com.amplitude:android-sdk`: Maintenance Android SDK
+* `com.amplitude:analytics-android`: Latest Android SDK
+
+## Dependency
+
+Update build.gradle to remove the maintennace Android SDK and add the latest Android SDK. Then sync project with Gradle files.
+
+```diff
+dependencies {
+-  implementation 'com.amplitude:android-sdk:2.+'
+-  implementation 'com.squareup.okhttp3:okhttp:4.2.2'
++  implementation 'com.amplitude:analytics-android:1.+'
+}
+```
+
+## Instrumentation
+
+The latest Android SDK offers an new API to instrument events. To migrate to it, you need to update a few calls. The following sections detail which calls have changed.
+
+### Initialization
+
+You must initialize the SDK with a valid Amplitude API Key and Android application context.
+
+=== "Kotlin"
+
+    ```diff
+    - import com.amplitude.api.Amplitude
+    - import com.amplitude.api.AmplitudeClient
+    + import com.amplitude.android.Amplitude
+
+    - val client = Amplitude.getInstance()
+    -   .initialize(getApplicationContext(), "YOUR_API_KEY")
+    + val client = Amplitude(
+    +     Configuration(
+    +         apiKey = "YOUR_API_KEY",
+    +         context = getApplicationContext()
+    +     )
+    + )
+    ```
+
+=== "Java"
+
+    ```diff
+    - import com.amplitude.api.Amplitude;
+    - import com.amplitude.api.AmplitudeClient;
+    + import com.amplitude.android.Amplitude;
+
+    - AmplitudeClient client = Amplitude.getInstance()
+    -   .initialize(getApplicationContext(), "YOUR_API_KEY");
+    + Amplitude client =  new Amplitude(new Configuration(
+    +     apiKey = "YOUR_API_KEY",
+    +     context = getApplicationContext()
+    + ));
+    ```
+
+### Configuration
+
+The latest Android SDK configuration comes in a different shape. Some configurations are no longer supported.
+
+|com.amplitude:android-sdk|com.amplitude:analytics-android|
+|-|-|
+| `eventUploadPeriodMillis` | `flushIntervalMillis` |
+| `eventUploadThreshold` | `flushQueueSize` |
+| `eventUploadMaxBatchSize` | Not supported |
+| `eventMaxCount` | Not supported |
+| `identifyBatchIntervalMillis` | `identifyBatchIntervalMillis` |
+| `flushEventsOnClose` | `flushEventsOnClose` |
+| `optOut` | `optOut` |
+| `trackingSessionEvents` | `trackingSessionEvents` |
+| `sessionTimeoutMillis` | Not supported |
+| `minTimeBetweenSessionsMillis` | `minTimeBetweenSessionsMillis`. The maintenance SDK disables foreground tracking by default and uses `sessionTimeoutMillis` when foreground tracking is disabled. `minTimeBetweenSessionsMillis` is used when foreground tracking is enabled by `enableForegroundTracking()`. However, the latest SDK enables foreground tracking and it's not configurable.|
```

Comment:
I feel the content of `sessionTimeoutMillis` is more suitable to the `sessionTimeoutMillis` section. wdyt?
```suggestion
| `sessionTimeoutMillis` | Not supported.  The maintenance SDK disables foreground tracking by default and uses `sessionTimeoutMillis` when foreground tracking is disabled. However, the latest SDK enables foreground tracking and it's not configurable. |
| `minTimeBetweenSessionsMillis` | `minTimeBetweenSessionsMillis`|
```

Reply:
Yes! it's more clear